### ISSUE TITLE
Right-to-left languages subtitles having punctuation in the wrong order.

### ISSUE THREAD

**Issue by pypp** [jellyfin/jellyfin-web#4179]
**Describe The Bug**
Right-to-left languages subtitles having punctuation in the wrong order.

**Screenshots**

The problem is due to current web browsers rendering the every VTT file in the same way and ingonring right to left.

![Screenshot 2022-11-19 205059](https://user-images.githubusercontent.com/55145572/202866849-221d330e-f60e-478e-b579-84e71d3454d2.png)

The subtitles should be renders like this (with the punctuation to the left side); 

> וארבעה ילדים אשר...

**Comment by thornbill** [comment]
RTL languages are not properly supported in 10.8 and earlier. Have you tested this in an unstable build?

**Comment by 1netanel** [comment]
I'm using jellyfin version: 10.8.7

**Comment by ahbanavi** [comment]
Same problem with Persian...

**Comment by thornbill** [comment]
This needs testing in unstable to confirm it is fixed on master.

**Comment by Ignio-dev** [comment]
Hey,
Continuing our discussion in Discord-Matrix:
1. This issue still occures on master and occured on 10.8 as well as latest.
2. There is no difference whether the UI language is set to be the same rtl language.
3. An example subtitle file is attached here (it ends with an srt normally, just changed it's extension because github doesn't accept uploading of .srt files): [yellowstone.us.s01e01.1080p.bluray.x264-rovers.he.txt](https://github.com/jellyfin/jellyfin-web/files/12895546/yellowstone.us.s01e01.1080p.bluray.x264-rovers.he.txt)
4. An example from all current jellyfin versions: 
<img width="960" alt="incorrect" src="https://github.com/jellyfin/jellyfin-web/assets/57571508/31055d5d-ed75-448c-8e2e-013793c38e2f">
5. An example when pull request #3711 is applied:
<img width="960" alt="correct" src="https://github.com/jellyfin/jellyfin-web/assets/57571508/967ee9a2-5adc-452a-97c6-59e0cfdcd8db">


If the fix offered in pull request #3711 isn't the correct way to go about, I would appreciate if you could point me in the right direction to fix it.
Thanks!

**Comment by Shasoosh** [comment]
Plex is suffering from the same problem for years.
I'd love to see this fixed in jellyfin.

**Comment by thornbill** [comment]
I tried a few different things but the only thing that seemed to work for me was the approach in the closed PR. I added that modification to the function that normalizes other issues in subtitles. If you would like to confirm that it is working, you can use the preview url linked in the PR: https://github.com/jellyfin/jellyfin-web/pull/4895#issuecomment-1771688772

I'll put a call out for testing in Matrix also since it could possibly break subtitles if they were working without the change.

**Comment by pypp** [comment]
Unfortunately i my jellyfin instance isn't exposed to the internet, so I can't test it. maybe someone else in this thread can test it.

**Comment by pypp** [comment]
Just tried it, and it works great.
